.github/workflows/integration-tests-release.yaml

```diff
@@ -0,0 +1,219 @@
+---
+name: Integration Tests and Release
+
+on: 
+  pull_request:
+    paths:
+      - 'pkg/**'
+      - 'cmd/**'
+      - 'charts/**'
+      - 'tests/integration/**'
+      - '.github/workflows/**'
+  push:
+    branches:
+      - 'main'
+      - '**-run-ci'
```

Comment:
Didn't we opt to not creating any non-fork branches for features? What are these for?

Reply:
This allows the opt-in approach to running CI when pushing commits without a PR. That's what we agreed upon.

Reply:
But that was for forks only, not k8ssandra-repo?

Reply:
I'm not sure how to make that distinction